modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/organization/management/v1/OrganizationManagementSuccessTest.java

```diff
@@ -1135,6 +1139,208 @@ public void testDeleteOrganizationsForPagination() {
         Assert.assertTrue(organizations.isEmpty(), "All organizations should be deleted, but the list is not empty.");
     }
 
+    @Test(groups = "organizationMetaAttributesPaginationTests", dependsOnMethods =
+            "testDeleteOrganizationsForPagination")
+    public void createOrganizationsForMetaAttributesPaginationTests() throws JSONException {
+
+        organizations = createOrganizations(1);
+        assertEquals(organizations.size(), 1);
+
+        organizationForMetaAttributes = organizations.get(0);
+        assertNotNull(organizationForMetaAttributes);
+
+        organizations.clear();
+    }
+
+    @Test(groups = "organizationMetaAttributesPaginationTests",
+            dependsOnMethods = "createOrganizationsForMetaAttributesPaginationTests")
+    public void testAddMetaAttributesToOrganizations() {
+
+        // Sorted in the order.
+        String[] attributes =
+                {"1", "2", "3", ":", "@", "A", "B", "C", "LMN", "PQR", "STU", "a", "b", "c", "fg", "jKL", "mNo", "x",
+                        "y", "z"};
+        metaAttributes = new ArrayList<>();
+
+        String organizationId = organizationForMetaAttributes.get(ORGANIZATION_ID);
+        assertNotNull(organizationId, "Organization ID should not be null");
+
+        for (String attribute : attributes) {
+            String requestBody =
+                    "[{\"operation\":\"ADD\",\"path\":\"/attributes/" + attribute + "\",\"value\":\"value-" +
+                            attribute + "\"}]";
+
+            String endpointURL = ORGANIZATION_MANAGEMENT_API_BASE_PATH + "/" + organizationId;
+            Response response = getResponseOfPatch(endpointURL, requestBody);
+
+            validateHttpStatusCode(response, HttpStatus.SC_OK);
+
+            // Save the attributes to the instance variable.
+            Map<String, String> attrMap = new HashMap<>();
+            attrMap.put(ORGANIZATION_ID_ATTRIBUTE, organizationId);
+            attrMap.put(ORGANIZATION_META_ATTRIBUTE_ATTRIBUTE, attribute);
+            metaAttributes.add(attrMap);
```

Comment:
Do we need a map to store the meta attributes, since all of them are under one organization ID? We could simply store them in a string array.

Reply:
Addressed with https://github.com/wso2/product-is/pull/21054/commits/7f95c8efe6686a47acfec90aa0973af6e6e3bcc7